lasagne/layers/helper.py

```diff
@@ -290,9 +290,10 @@ def get_output_shape(layer_or_layers, input_shapes=None):
         return all_shapes[layer_or_layers]
 
 
-def get_all_params(layer, **tags):
+def get_all_params(layer, unwrap_shared=True, **tags):
     """
-    Returns a list of Theano shared variables that parameterize the layer.
+    Returns a list of Theano shared variables or expressions that
+    parameterize the layer.
 
     This function gathers all parameter variables of all layers below one or
```

Comment:
@wielandbrendel, as we just discussed, could you change "parameter variables" to "parameters" here, amending your commit or squashing afterwards? Thanks! (I don't feel fully comfortable doing this as part of the merge, I think github will not render the PR as "merged" but as "closed" then. I should try that sometime.)